src/main/java/uk/gov/digital/ho/hocs/info/api/TopicResource.java

```diff
@@ -43,6 +43,7 @@ public TopicResource(TopicService topicService, ParentTopicRepository parentTopi
         return ResponseEntity.ok(GetAllTopicsResponse.fromTopicWithFilteredChildren(parentTopics, filteredChildTopics));
     }
 
+    // todo: remove
```

Comment:
Does this need removed?

Reply:
Yeah... I put it there to remind me that the method needs to be removed however I didn't want to do it. I can though